src/App.vue

```diff
@@ -751,7 +751,8 @@ export default class App extends Mixins(
       submitter: header.submitter,
       data: {
         applicationDate: this.dateToYyyyMmDd(this.apiToDate(header.date)),
-        legalFilings: [header.name]
+        legalFilings: [header.name],
+        order: data.courtOrder
```

Comment:
Because this was missing, bootstrap filings didn't have court order info to display.

(Bootstrap filings are paid IAs/Amalgamations/Continuation Ins. They are still referenced using a Temporary Registration Number. This code makes the "filings" response look like a ledger list.)

(If bootstrap filings are in a state before Paid -- for example, Pending -- then they are still in the Todo List.)